cmd/envbuilder/main.go

```diff
@@ -25,12 +24,21 @@ import (
 )
 
 func main() {
+	cmd := envbuilderCmd()
+	err := cmd.Invoke().WithOS().Run()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v", err)
+		os.Exit(1)
+	}
+}
+
+func envbuilderCmd() *serpent.Command {
```

Comment:
Nit: Could probably skip the ptr here. 🤔

Also: Good choice splitting this up.